src/components/token/TokenAlerts.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState, useEffect } from 'react';
+
+
+const TokenAlerts = (props) => {
+  const [token, setToken] = useState(props.token);
+
+  useEffect(() => {
+    setToken(props.token);
+  }, [props.token]);
+
+  const bannedAlert = () => {
+    if (!token.banned) {
+      return null;
+    }
+
+    return (
+      <div className="alert alert-danger backup-alert" role="alert">
+        <i className="fa fa-exclamation-triangle mr-2" title="Token Verified"></i>
+        This token was banned and should not be trusted. <a href="http://hathor.network" >Learn more.</a>
```

Comment:
This is the message that is being discussed? Maybe we should leave a TODO only instead of this message. Besides that, would be better if the href link were for https instead of http, right?

Reply:
Moved to a const and added a todo